spec/integration/middleware_spec.rb

```diff
@@ -27,7 +27,7 @@ def app
       get '/rack-mini-profiler/requests', {}, 'HTTP_ACCEPT_ENCODING' => 'gzip, compress'
 
       expect(last_response.body).to include('<title>Rack::MiniProfiler Requests</title>')
-      expect(last_response.body).to match('<body>\n  </body>')
+      expect(last_response.body).to match('<body>\n  <script async nonce="" type="text/javascript" id="mini-profiler"')
```

Comment:
This is interesting. Without the .dup, it looks like we were not inserting the rmp script?